addons/website/static/src/scss/website.scss

```diff
@@ -899,6 +901,13 @@ $-o-carousel-controllers-size: var(--CarouselControllers-size, #{map-get($spacer
         }
     }
 
+    // Make the entire slide clickable by disabling pointer events on the
+    // carousel control's wrapper, while keeping the controls themselves
+    // clickable.
+    .carousel-indicators button, .carousel-control-prev, .carousel-control-next {
```

Comment:
Even if I understand it: Confusing comment, it's not this rule that "Make the entire slide clickable".
Also don't target "button", we have another PR where we might go back to `<a>` instead of button because of sanitization issue. Can't you target something like `> *`? I suppose they are direct children?